apps/marketplace/pages/__snapshots__/CreateUserPage.test.js.snap

```diff
@@ -28,7 +28,7 @@ exports[`Test suite for CreateUserPage page Because snaphots 1`] = `
           onSubmit={[Function]}
         >
           <div
-            className="field"
+            className="field "
```

Comment:
Like the password reset, did we miss updating these snaps?

Reply:
this is correct because of updates to text field